### ISSUE TITLE
Optional row-centric visualization (instead of column-centric)

### ISSUE THREAD

**Issue by novoid** [kiwanami/emacs-calfw#22]
Hi!

Most of my agenda entries are much longer than high. Therefore I would love to see an alternative style for example for a week where Monday is the first row (instead of the first column) in the layout. Tuesday is the row below and so forth. In contrast to the usual weekly agenda, I would love to see a second column with either next week or the second half of the current week (see image below).

My content could be displayed _much_ better since with the current calfw, I only see the Org-mode file and probably the first two words of each entry - which is not of any help in most cases.

I attached an example sketch of a view I would like to use:

![2013-01-14T10 10 Screenshot - calfw week view sketch](https://f.cloud.github.com/assets/727493/64060/4fddd6f8-5e2a-11e2-97c2-ef022eb9dd99.png)

So it's more or less the same way as now but in a transposed form.

**Comment by novoid** [comment]
Hi kiwanami!

I just want to mention that I met a couple of guys desperately looking for a cool Org-agenda visualization like yours. However, all of them said that this would only be of any use with the horizontal layout I suggested above.

If you find any spare time, this would be a great improvement for lots of people out there!

**Comment by kiwanami** [comment]
Hi @novoid 
Thank you for your comment. Now I'm trying to implement recurring events.
Then, I'll try to do this.

**Comment by pasja** [comment]
@kiwanami : if you need tester for the recurring events feature, let me know!

**Comment by kiwanami** [comment]
@pasja Thank you for your offer!
I'll update the status at #8.

**Comment by mankoff** [comment]
@novoid single column view can be achieved with Org agenda views. Is the 2nd column important? If not, consider a custom agenda and not using calfw for this.

**Comment by skyer9** [comment]
custom row count
custom cell count
custom day count to display
custom agenda item line format...

**Comment by novoid** [comment]
@mankoff I am using single column view at the moment. However, I'd love to use calfw with dual column view as shown in mock-up above. Can't be done via standard agenda views.

**Comment by mankoff** [comment]
@novoid Yes, I agree, Org doesn't do dual column. I would like that too, and hope this feature does get implemented.

As for @skyer9, I'm not sure what the word list above means, and I'm not trying to push Org agendas when they aren't the right view, but in Org Agenda you can set the number of items shown each day (custom row count?), set the number of days shown (custom cell count?), customize the format of each line, etc.

**Comment by skyer9** [comment]
it needs vertical/horizontal count for day grid view.

total days to show in day grid view.

and, custom agenda item line format.

i think i can add variable for "custom agenda item line format".

**Comment by skyer9** [comment]
with this config, user can make two week view.

for example,
vertical count : 7
horizontal count : 2
days count : 14

**Comment by novoid** [comment]
@skyer9 You're talking about a possible feature for the future, right?

Or is this existing functionality? If so, where can I find its documentation?

**Comment by skyer9** [comment]
yes, feature for the future.

and i will do on functionality for "custom agenda item line format" in this weekend.

**Comment by novoid** [comment]
@skyer9 Ah. Thanks!

**Comment by kiwanami** [comment]
I'm sorry for my late response and work...

**Comment by mankoff** [comment]
@kiwanami please don't feel bad or apologize. This is Open source software based on your donated time. It is great as-is, and will get better. If people need things now, they can code it themselves or pay others to implement it.

**Comment by novoid** [comment]
@kiwanami and @mankoff: I totally agree: don't feel bad or pressed! It's such a great thing that you invest time for an open source project!

**Comment by kiwanami** [comment]
@mankoff @novoid Thank you for your comment!
Recently, I have been getting busier because my job slightly changed.
However, I'd continue to attend to my works and emacs projects as much as possible.

**Comment by bepolymathe** [comment]
Hi, any news about this ?

**Comment by novoid** [comment]
https://github.com/ml729/calfw-blocks has implemented this view as described in #112
